src/rosdiscover/launch.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+__all__ = ('Launch',)
+
+from typing import Any, Mapping, List
+
+import attr
+
+
+# Data class storing the file name of a launch file and its arguments
```

Comment:
move this description into the docstring on line 12

Reply:
done